app/src/org/commcare/adapters/MenuAdapter.java

```diff
@@ -135,9 +135,6 @@ private void addRelevantCommandEntries(Menu m, Vector<MenuDisplayable> items,
                     Logger.log(AndroidLogger.TYPE_ERROR_CONFIG_STRUCTURE, msg);
                     throw new RuntimeException(msg);
                 }
-                if (!XPathFuncExpr.toBoolean(ret)) {
-                    continue;
-                }
```

Comment:
Removing this because I have no idea what it would do given https://github.com/dimagi/commcare-odk/pull/1307/files#diff-bc3a12589746386b2d565b70c55f4adfL129